snuba/subscriptions/consumer.py

```diff
@@ -10,7 +10,7 @@
 )
 
 from snuba.utils.streams.consumer import Consumer
-from snuba.utils.streams.types import Message, Partition, Topic
+from snuba.utils.streams.types import ConsumerError, Message, Partition, Topic
```

Comment:
`ConsumerError` probably should have gone in `snuba.utils.streams.consumer` and not `types`, not sure what I was thinking here.